ember-addon-main.js

```diff
@@ -46,6 +47,20 @@ module.exports = {
   included: function (app) {
     this._super.included.apply(this, arguments);
 
+    app.options = app.options || {};
+    app.options.babel = app.options.babel || {};
+    app.options.babel.plugins = app.options.babel.plugins || [];
+
+    var Compiler = require(this.templateCompilerPath());
```

Comment:
this should be:

```js
var Compiler = this.htmlbarsOptions().templateCompiler.compile
```

Once ^^ is done, the precompiler will have access to the AST transforms